sdks/java/core/src/main/java/org/apache/beam/sdk/values/ShardedKey.java

```diff
@@ -16,15 +16,15 @@
  * limitations under the License.
  */
 
-package org.apache.beam.sdk.io.gcp.bigquery;
+package org.apache.beam.sdk.values;
```

Comment:
Last time I suggested doing this, @dhalperi suggested that ShardedKey belonged in values.